### PR TITLE
Rename Open Media Streamer(OMS) to Open WebRTC Toolkit(OWT).

### PR DESCRIPTION
**Pull request by jianjunz**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ Run `grunt debug` in "scripts" folder and get files in "dist/sdk-debug".
 Run `grunt jsdoc` in "scripts" folder and get docs in "dist/docs".
 
 ## How to contribute
-We warmly welcome community contributions to Open Media Streamer JavaScript SDK repository. If you are willing to contribute your features and ideas to OMS, follow the process below:
+We warmly welcome community contributions to Open WebRTC Toolkit JavaScript SDK repository. If you are willing to contribute your features and ideas to OMS, follow the process below:
```
Comment (yanbin7) [inline README.md]: OMS should be OWT here

File: test/p2ptest/js/config.js [hunk 2]
```diff
@@ -1,4 +1,4 @@
-var serverIP = "place_holder";
+var serverIP = "http://10.239.196.20:8095/";
```
Comment (yanbin7) [inline test/p2ptest/js/config.js]: Why use static server address here ?

Comment (jianjunz) [reply test/p2ptest/js/config.js]: Reverted to `place_holder`.

File: README.md (hunk 1 above)
Comment (jianjunz) [reply README.md]: Fixed.

**Pull request merged**